css-pseudo-classes/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="styles.css" />
+    <title>css-pseudo-classes</title>
+  </head>
+  <body>
+    <div>
+      <section>
+        <h1>Hover button</h1>
+        <form>
+          <button type="button" class="block">I'm a button!</button>
+        </form>
```

Comment:
No `form` element is required here. We aren't submitting any data, so we can just use the `button` directly.